tests/test_main.py

```diff
@@ -95,7 +99,9 @@ def test_ruleset():
                     rule:
                         meta:
                             name: file rule
-                            scope: file
+                            scopes:
+                                static: file
+                                dynamic: process
```

Comment:
- [ ] double check dynamic scopes here, for most tests it's probably not relevant, but the static/dynamic pairs should still make sense